te_chef.py

```diff
@@ -172,22 +177,52 @@ def scrape_content(title, content_url):
         youtube_url = doc.select_one(".video-container iframe")["src"]
         youtube_id = get_youtube_id_from_url(youtube_url)
 
-        # Some of the videos have been removed from the YouTube channel --
-        # skip creating content nodes for them entirely so they don't show up
-        # as non-loadable videos in Kolibri.
         try:
-            ydl.extract_info(youtube_url, download=False)
+            info = ydl.extract_info(youtube_url, download=False)
+            subtitle_languages = info["subtitles"].keys()
+            print ("      ... with subtitles in languages:", subtitle_languages)
         except youtube_dl.DownloadError as e:
-                print("        NOTE: Skipping video download due to error: ", e)
-                return None
+            # Some of the videos have been removed from the YouTube channel --
+            # skip creating content nodes for them entirely so they don't show up
+            # as non-loadable videos in Kolibri.
+            print("        NOTE: Skipping video download due to error: ", e)
+            return None
 
         video_node = nodes.VideoNode(
             **base_node_attributes,
             derive_thumbnail=True,
             files=[files.YouTubeVideoFile(youtube_id=youtube_id)],
         )
 
-        # TODO(davidhu): Get subtitles in other languages
+        # Add subtitles in whichever languages are available.
+        for language in subtitle_languages:
+            if not languages.getlang(language):
+                # It seems like the subtitle language codes that we get as queried
+                # from youtube-dl are not all consistent with the codes of
+                # the languages in le-utils.
+                #
+                # E.g. we may get "zh-Hans" from
+                # youtube-dl but languages.getlang("zh-Hans") returns None
+                # while languages.getlang("zh") returns something.
```

Comment:
It seems `zh-Hans` is not a standard code, but something specific to YouTube. Maybe we can fallback to just `zh` for language codes that don't exist in `le_utils` or `iso-639`?

As in, if no `zh` subs exist, then add the `zh-unknown` as `zh` subs.